packages/@react-spectrum/s2/src/Toast.tsx

```diff
@@ -296,7 +296,10 @@ const toastContent = style({
   alignItems: 'baseline',
   gridArea: 'content',
   cursor: 'default',
-  width: 'fit'
+  width: 'fit',
+  overflowWrap: 'break-word',
```

Comment:
i think we only need this one? https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap